habitat/core/batched_env.py

```diff
@@ -455,6 +483,11 @@ def __init__(
         self.infos: List[Dict[str, Any]] = [{}] * self._num_envs
         self._previous_state: List[Optional[Any]] = [None] * self._num_envs
         self._previous_target_position = [None] * self._num_envs
+        self._stagger_agents = [0] * self._num_envs
```

Comment:
I don't understand the role of stagger agents, is this still needed?

Reply:
It is still needed, especially if the only way to end an episode is be succeed. See this post : https://cvmlp.slack.com/archives/C035YBRFGFL/p1650047936528419?thread_ts=1650047657.366689&cid=C035YBRFGFL 
Basically, the very first episode of each agent will not be of the full length but shorter so that they do not all end their episode at the same time.